tests/e2e/linalg_ext_ops/attention.mlir

```diff
@@ -66,6 +66,48 @@ func.func @causal_attention1x3x4() {
   return
 }
 
+func.func @attention1x4x4_i1_mask_all_ones() {
+  %init = tensor.empty() : tensor<1x4x4xf32>
+  %query = util.unfoldable_constant dense<[[[0.1, 0.2, 0.3, 0.4],
+                                            [0.5, 0.6, 0.7, 0.8],
+                                            [0.9, 1.0, 1.1, 1.2],
+                                            [1.3, 1.4, 1.5, 1.6]]]> : tensor<1x4x4xf32>
+
+  %key = util.unfoldable_constant dense<[[[0.1, 0.2, 0.3, 0.4],
+                                          [0.5, 0.6, 0.7, 0.8],
+                                          [0.9, 1.0, 1.1, 1.2],
+                                          [1.3, 1.4, 1.5, 1.6]]]> : tensor<1x4x4xf32>
+  %value = util.unfoldable_constant dense<[[[0.1, 0.2, 0.3, 0.4],
+                                            [0.5, 0.6, 0.7, 0.8],
+                                            [0.9, 1.0, 1.1, 1.2],
+                                            [1.3, 1.4, 1.5, 1.6]]]> : tensor<1x4x4xf32>
+
+  %mask = util.unfoldable_constant dense<[[[true, true, true, true],
+                                           [true, true, true, true],
+                                           [true, true, true, true],
+                                           [true, true, true, true]]]> : tensor<1x4x4xi1>
```

Comment:
For unit testing, would it be useful to set some of these mask values to `false`?

Maybe add a new test case `attention1x4x4_i1_mask_some_ones`?

Reply:
There is already a test which has some mask values set to `false` in the same file, so I am not testing it for general cases. This all ones test has a copy in `attention_i1_mask.mlir`,  the purpose is to test that the results are exactly the same under both memory layout configurations.

Reply:
If we intend to set all values to true, why not just write something like `dense<true>`?